CONTRIBUTING.md

```diff
@@ -6,12 +6,21 @@ the section below we ask that the following guidelines be considered when
 opening, reviewing, and merging pull requests.
 
 * No one should merge their own code or code from their own organization
-* Reviewers should included people from an organization that is not your own
+* Reviewers should include people from an organization that is not your own
 * If no changes have been requested by other reviewers the (last) reviewer
 merges the pull request on approval
 * The CNCF [#tag-runtime](https://cloud-native.slack.com/archives/CPBE97SMU)
 Slack channel should be used for discussion
 
+## Community Meetings
+
+We have regular community meetings in place to discuss CDI-related topics.
+The current cadence of the meetings is bi-weekly. You can find the meeting
+details in the [CNCF Community Meetings Calendar](https://www.cncf.io/calendar).
+The calendar entry also has a link to our community meeting agenda. The agenda
+has a summary of past meeting topics and discussions. It can also be freely
+edited by anyone to propose topics for upcoming meetings.
```

Comment:
Should we also mention the name of the meeting as it shown at the link `[CNCF TAG Runtime Container Device Interface (COD) Working Group](https://tockify.com/cncf.public.events/detail/184/1719324000000?search=COD)`?

Reply:
Pointing to the meeting notes https://docs.google.com/document/d/1gUgAMEThkRt4RJ7pA7ZbPPmIOX2Vb7fwH025MjfcTYU would also make sense, I think.

Reply:
We discussed this and decided to deliberately not link directly to the meeting agenda, but instead mention that it is reachable via an up-to-date link through the meeting calendar entry. This way we don't need to worry about keeping the meeting agenda link up to date in the documentation if it ever changes.

And thanks for the tockify hint/link. I added now your suggested search link to our docs, so getting to the calendar should now be behind two clicks, the agenda behind 3 clicks, and no manual searching any more.